static-data/bibliotecas.json

```diff
@@ -276,6 +260,22 @@
                     "xml:lang": "pt"
                 }
             },
+            {
+                "geo": {
+                    "datatype": "http://www.opengis.net/ont/geosparql#wktLiteral",
+                    "type": "literal",
+                    "value": "Point(-9.17152 38.82741)"
+                },
+                "item": {
+                    "type": "uri",
+                    "value": "http://www.wikidata.org/entity/Q100871052"
+                },
+                "itemLabel": {
+                    "type": "literal",
+                    "value": "Biblioteca Muncipal Jos\u00e9 Saramago",
+                    "xml:lang": "pt"
+                }
+            },
```

Comment:
Esta entrada não deveria estar aqui, segundo a ordem alfabética 🤔

Reply:
Está escrito Muncipal em vez de Municipal. :)
Está errado mas é do lado do wikidata.

Não conheço o processo de atualizar daquele lado. Sabes fazer? 
Se a atualização for rápida, podes fazer e correr novamente o script, e assim fica aqui já com o nome certo?

Reply:
Well spotted! A atualização é simples, basta ir ao link do item, http://www.wikidata.org/entity/Q100871052, e carregar no botão de editar aqui:

![Screenshot 2020-10-26 at 14 54 31](https://user-images.githubusercontent.com/478237/97188349-521e0180-179b-11eb-8499-f02697290f88.png)

Após gravar, os dados ficam logo live, podendo haver um delay de alguns segundos para o query service o apanhar. Queres fazê-lo para testar as águas?

Reply:
Ah, fixe. Já alguém o fez, portanto está OK.
Corres outra vez o script e atualizas os commits? (se já o fizeste, diz)

Já agora, era fixe adicionar essa info ao README (eg: "vês algum dado errado? Como corrigir")

Reply:
> Corres outra vez o script e atualizas os commits? (se já o fizeste, diz)

Corri o script, mas há uma série de alterações incorretas, devido à mudança da label em Português de várias bibliotecas para "Biblioteca municipal" 🙁 [Exemplo 1](https://www.wikidata.org/w/index.php?title=Q72304657&diff=1297881194&oldid=1296732182), [Exemplo 2](https://www.wikidata.org/w/index.php?title=Q73790876&diff=1297888218&oldid=1296735632), [Exemplo 3](https://www.wikidata.org/w/index.php?title=Q71606077&diff=next&oldid=1296681385). @themudo sabes de alguma forma de reverter/corrigir essas alterações de forma mais prática do que à mão?

Para já fiz a alteração manualmente desta entrada, até os dados serem corrigidos no Wikidata.

> Já agora, era fixe adicionar essa info ao README (eg: "vês algum dado errado? Como corrigir")

Cria um issue please, eu curtia expandir o README (até do ponto de vista do #55) mas preferia esperar até o processo estabilizar (#53, #51, #48).

Reply:
Quanto a este PR, dei uma vista de olhos aos dados que foram atualizados e parecem-me bem. Por mim está pronto para ser merged.

Reply:
@waldyrious foi um erro que eu introduzi sem querer por falta de cuidado. Penso que já foram corrigidos todos os casos.

Reply:
@themudo confirmo, já estão todos corrigidos — faltava só um, mas [já o arranjei](https://www.wikidata.org/w/index.php?title=Q74201118&diff=1299118201&oldid=1297889495)! 😃

Reply:
>> Já agora, era fixe adicionar essa info ao README (eg: "vês algum dado errado? Como corrigir")
>
> Cria um issue please, eu curtia expandir o README (até do ponto de vista do #55) mas preferia esperar até o processo estabilizar (#53, #51, #48).

Criei o issue #63 para melhoria da documentação de contribuição, por isso vou marcar esta conversa como resolvida.